Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -293,20 +292,18 @@
      * the class during application runtime.</p>
      */
     private static Branch branchReferral_;
-    
+
     private BranchRemoteInterface branchRemoteInterface_;
-    private final PrefHelper prefHelper_;
+    final PrefHelper prefHelper_;
     private final DeviceInfo deviceInfo_;
     private final Context context_;
-    
+
     private final Semaphore serverSema_ = new Semaphore(1);
-    
+
     final ServerRequestQueue requestQueue_;
     
-    private int networkCount_ = 0;
-    
-    private boolean hasNetwork_ = true;
```

Comment:
Not needed anymore after making change to BranchPostTask (this flag was used to block subsequent requests when previous request failed)